content/vendor/guides/cli-quickstart.md

```diff
@@ -45,21 +47,20 @@ $ replicated version
 }
 ```
 
-#### Setting an API Token
+* * *
+
+#### 2. Setting an API Token
 
 Next we'll set up two environment variables. You can export these right in your shell or add them to your favorite dotfiles.
 
-```shell script
-export REPLICATED_APP=
-export REPLICATED_API_TOKEN=
-```
+If you haven't already, you'll want to head over to https://vendor.replicated.com and create an Application. You can name the app whatever you want, we'll use "CLI Quickstart" in this case. From there you can head to the settings page, grab the Application Slug, and export it:
```

Comment:
@codelingobot One sentence per line. Each new sentence should be written on a new line in Markdown files.

Reply:
thanks, will update

Reply:
@marccampbell CodeLingo does not currently support retroactively adding check runs to existing open PRs, hence why you did not get a response from the bot. Also, please prepend capture comments with "capture".

Reply:
@Daanikus for sure, thanks -- for now I can be the code bot 🙂